scripts/build_example

```diff
@@ -95,15 +94,16 @@ esac
 
 if [[ -n "${docker_config}" ]]; then
   "${SCRIPTS_DIR}/build_server" -s "${docker_config}"
-  if [[ "${EXAMPLE}" == "tensorflow" ]]; then
-    bazel --output_base="${CACHE_DIR}/emscripten" build "${bazel_build_flags[@]}" --config=emscripten //examples/tensorflow/server:tensorflow_image.tar
-  else
-    bazel build "${bazel_build_flags[@]}" "--config=clang" "//examples/${EXAMPLE}/server:${EXAMPLE}_image.tar"
-  fi
+  # Build the base server.
+  docker build -t oak_docker -f ./oak/server/Dockerfile ./oak/server
+  # Build and save the example
+  docker build -t "${EXAMPLE}_image" -f ./examples/Dockerfile "./examples/${EXAMPLE}"
```

Comment:
Why the `_image` suffix? All docker images are... images.

Reply:
I am just reusing the existing names. Happy to change it, but I think we use it in a lot of places. cc: @ipetr0v - any reason for using _image?

Reply:
Since it's an example that is being loaded by Oak server inside Docker, it also could be `"${EXAMPLE}_server"`.
But just `"${EXAMPLE}"` is also fine.